### PR TITLE
Add-167_two_sum_ii-input_array_is_sorted

### PR DESCRIPTION
**Pull request by Amoonguses1**
I solved two ways because I thought binary search is not effective.

### CODE REVIEW

**Comment by Amoonguses1** [comment]
Please check 167_two_sum_ii-input_array_is_sorted.py . this file solved problem with binary search.
@tkkawa @Ryuju0911

**Comment by Amoonguses1** [comment]
I have rewrote my code to follow Meguru style and PEP8.
@tkkawa @Ryuju0911

File: 167_two_sum_ii-input_array_is_sorted.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class Solution:
+
+    def twoSum(self, numbers: List[int], target: int) -> List[int]:
+        nums_length = len(numbers)
+        indices_list = []
+        for i in range(nums_length):
+            left = i
+            right = nums_length
+            while left < right - 1:
+                mid = (left + right) // 2
```
Comment (tkkawa) [inline 167_two_sum_ii-input_array_is_sorted.py:15]: See [Pep8](https://peps.python.org/pep-0008/).

File: 167_two_sum_ii-input_array_is_sorted_twopointers.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    def twoSum(self, numbers: List[int], target: int) -> List[int]:
+        left = 0
+        right = len(numbers) - 1
+        answer_list = []
+        while left != right:
+            if numbers[left] + numbers[right] == target:
+                answer_list.append(left+1)
+                answer_list.append(right+1)
+                return answer_list
```
Comment (tkkawa) [inline 167_two_sum_ii-input_array_is_sorted_twopointers.py:16]: Insert one line after the return statement.

**Review by tkkawa — COMMENTED** [review]
You can write two solutions in one file.

**Comment by Amoonguses1** [comment]
I add two pointers solution to binary search solution file. I fixed you pointed out.
@tkkawa

**Comment by tkkawa** [comment]
See [the repository](https://github.com/okuda-seminar/leetcode_answers).

**Comment by Amoonguses1** [comment]
I added error handling and docstring. Please check my code.
@tkkawa

**Review by tkkawa — APPROVED** [review]
@Amoonguses1 
LGTM

**Pull request merged**